crates/pop-parachains/src/new_parachain.rs

```diff
@@ -75,12 +80,72 @@ pub fn instantiate_standard_template(
 	Ok(tag)
 }
 
+/// Create a new Tanssi compatible container chain.
+///
+/// # Arguments
+///
+/// * `template` - template to generate the parachain from.
+/// * `target` - location where the parachain will be created.
+/// * `tag_version` - version to use (`None` to use latest).
+fn instantiate_tanssi_template(
+	template: &Parachain,
+	target: &Path,
+	tag_version: Option<String>,
+) -> Result<Option<String>> {
+	let temp_dir = ::tempfile::TempDir::new_in(std::env::temp_dir())?;
+	let source = temp_dir.path();
+	let tag = Git::clone_and_degit(template.repository_url()?, &source, tag_version)?;
+
+	// Relevant files to extract.
+	let files =
+		[".rustfmt.toml", "Cargo.toml", "Cargo.lock", "LICENSE", "README.md", "rust-toolchain"];
+	for file in files {
+		extract_template_files(file.to_string(), temp_dir.path(), target, None)?;
+	}
+
+	let owned_target = target.to_owned().to_path_buf();
+
+	// Templates are located in Tanssi's repo as follows:
+	// │
+	// ├┐ container-chains
+	// │├┬ nodes
+	// ││└ <template node directories>
+	// │└┬ runtime-templates
+	// │  └ <tempalte runtime directories>
+
+	// Step 1: extract template node.
+	let template_path = format!("container-chains/nodes/{}", template.to_string());
+	extract_template_files(
+		template_path,
+		temp_dir.path(),
+		&owned_target.join("node").as_path(),
+		None,
+	)?;
+	// Step 2: extract template runtime.
+	let template_path = format!("container-chains/runtime-templates/{}", template.to_string());
+	extract_template_files(
+		template_path,
+		temp_dir.path(),
+		&owned_target.join("runtime").as_path(),
+		None,
+	)?;
+
+	// Add network configuration.
+	use askama::Template;
+	let network = Network { node: format!("container-chain-{}-node", template.to_string()) };
+	write_to_file(&target.join("network.toml"), network.render().expect("infallible").as_ref())?;
```

Comment:
This might need a zn config template specific for container chains so it includes a tanssi node chain by default. That we can then source and run.

Tanssi release not only includes `tanssi-node` but the template nodes we are adding in this PR.

Reply:
Do they provide cross-platform builds? If not we can possibly provide, but really need to improve the automation of updates there as it's still manual triggers.

Reply:
We might need to provide at least `apple-darwin`

Reply:
To be addressed in the following PR implementing the deployment logic.